fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/Select.java

```diff
@@ -249,8 +268,18 @@ public String toDebugString() {
      * @param renderer
      * @return
      */
+    @SuppressWarnings("unchecked")
     public <T> T render(StatementRenderer<T> renderer) {
```

Comment:
Is T ever NOT a string?

Reply:
T is going to be whatever the renderer returns (currently only have a String implementation of the renderer, but I guess could potentially have others). I rewrote this so the renderer handles the union - so not casting to String. I removed the SuppressWarnings.